hivemind/utils/grpc.py

```diff
@@ -208,3 +223,28 @@ def combine_from_streaming(stream: Iterable[runtime_pb2.Tensor]) -> runtime_pb2.
         buffer_chunks.append(tensor_part.buffer)
     serialized_tensor.buffer = b"".join(buffer_chunks)
     return serialized_tensor
+
+
+RpcMessage = TypeVar("RpcMessage")
+
+
+async def gather_from_grpc(
```

Comment:
When this PR is ready, is it true that we won't use grpc anywhere and this function gonna disappear?

Reply:
yes. You want to wipe out  `grpc` from the all namings?

Reply:
At least from this one, since it's introduced in this PR :)

Also, is it true that we can generalize this and [accumulate_parts_streaming](https://github.com/learning-at-home/hivemind/blob/master/hivemind/averaging/allreduce.py#L335) in some way, since they both deserialize a series of tensors received over the network asynchronously?

Reply:
While we've agreed with @justheuristic that wiping out all the grpc stuff can be done in a future PR, let's avoid introducing new grpc stuff here :)

Reply:
Honestly I don't know right now, because I didn't touch `averaging` in this PR, so I don't know much about it.
On the first glance `_accumulate_parts_streaming` does something more complex than just gathering parts together